src/Mechanisms/HandleComponents/Synthesizers/Tests/FloatFormSynthUnitTest.php

```diff
@@ -0,0 +1,45 @@
+<?php
+
+namespace Livewire\Mechanisms\HandleComponents\Synthesizers\Tests;
+
+use Livewire\Component;
+use Livewire\Form;
+use Livewire\Livewire;
+
+class FloatFormSynthUnitTest extends \Tests\TestCase
+{
+    /**
+     * @test
+     */
+    public function null_value_hydrated_returns_null()
+    {
+        Livewire::test(FormComponentWithNullableFloat::class)
+            ->set('form.floatField', null)
+            ->assertSet('form.floatField', null, true); // Use strict mode
```

Comment:
There is now an `assertSetStrict('form.floatField, null);` method you can use which is descriptive meaning we can get rid of all the strict mode comments 🙂

Reply:
Done in commit [feb8cb8](https://github.com/livewire/livewire/pull/8126/commits/feb8cb8c0d7c71fcae4fa4f0c7f517e78d9ab33a)